core/templates/dev/head/css/oppia.css

```diff
@@ -607,10 +603,6 @@ textarea {
   Styles for the link to the site feedback form.
 */
 .oppia-site-feedback {
-  background-color: #fff;
```

Comment:
Hi @Rudy-Menekam, I do not think this is a correct fix. These settings were added to ensure that site feedback button is placed correctly (#7869). So, I am closing this PR as of now. Please feel free to pick up another issue since there is PR #7904 opened as a fix.